Sources/MapLibreSwiftUI/MapViewCoordinator.swift

```diff
@@ -268,6 +268,10 @@ public class MapViewCoordinator<T: MapViewHostViewController>: NSObject, MLNMapV
     }
 
     func addLayers(to mglStyle: MLNStyle) {
+        let firstSymbolLayer = mglStyle.layers.first { layer in
+            layer is MLNSymbolStyleLayer
+        }
```

Comment:
Highlighting for discussion. I went for the simplest implementation here. I don't think that there should be much overhead in computing this every time, since this should be a cheap check. But it's technically not needed every time (there may not always be a style layer with the `belowSymbols` ordering).

If anyone is concerned about this, we could make this lazy with a bit more code.